airflow/auth/managers/simple/simple_auth_manager.py

```diff
@@ -87,9 +85,13 @@ def get_generated_password_file() -> str:
             "simple_auth_manager_passwords.json.generated",
         )
 
+    @staticmethod
+    def get_users() -> list[list[str]]:
+        users = conf.getlist("core", "simple_auth_manager_users", " ")
+        users = [user.split(",") for user in users]
+        return users
```

Comment:
```suggestion
    @staticmethod
    def get_users() -> list[dict[str, str]]:
        users = [u.split(",", for u in conf.getlist("core", "simple_auth_manager_users")]
        return [{"username": username, "role": role} for username, role in users]
```

Don't need a fallback for `getlist()`, there is a default in `config.yml` already.

Something like this might be better. Instead of a list, keeps a dict which provides better context as to what the value is. It also better validates we get what we expect out of the config (no extra items, has a role). Could even expand this by validating the roles folks set too. This cuts down on some other changes in this PR too.

nit: `user:role` delimited by `,` feels more natural to me.

Reply:
Alternatively, you can use json in config to! Even multiline works (which I didn't know worked until recently).